llvm/unittests/Frontend/HLSLRootSignatureDumpTest.cpp

```diff
@@ -56,7 +56,7 @@ TEST(HLSLRootSignatureTest, DescriptorUAVClauseDump) {
   Clause.NumDescriptors = 3298;
   Clause.Space = 932847;
   Clause.Offset = 1;
-  Clause.Flags = DescriptorRangeFlags::ValidFlags;
+  Clause.Flags = llvm::dxbc::DescriptorRangeFlags(0x1000f);
```

Comment:
Should we `or` enum values together here as well? Also applies to a couple of places later in this file.